### DISCUSSION TITLE
i18n notification message

### DISCUSSION THREAD

**Discussion by EugeneKostin** [refinedev/refine#3325] [Q&A] [answered]
Hi everyone,
I use default i18n namespace instead "common" and find untranslated notifications. In the refine-core src files, the useList hook notification uses a message with a common namespace when other hooks use the default. Is this an issue or am I missing something?

useList
```
...
message: translate(
    "common:notifications.error",
    { statusCode: err.statusCode },
    `Error (status code: ${err.statusCode})`,
),
```

useOne
```
...
message: translate(
    "notifications.error",
    { statusCode: err.statusCode },
    `Error (status code: ${err.statusCode})`,
),
```
Thanks for your help!

**Comment by salihozdemir** [comment, marked as answer]
Hey @EugeneKostin,

It seems to be a issue. Do you want to open a PR for fixing this? Also, `useCustom` and `useCustomMutation` hooks like `useList` have similar problem. If you fixed all of them, we would pleasure 🚀

**Reply by omeraplak** [reply]
Hey @salihozdemir ,
I've created #3328 💥
